### DISCUSSION TITLE
The content of a field in a Firebase collection does not shows up in the Rowy table

### DISCUSSION THREAD

**Discussion by jnnptr** [AntlerVC/firetable#1239] [Support / Q&A] [answered]
What can be causing that a Rowy table is not showing the content of a field in the corresponding Firebase Collection? 

I created a table in Rowy, that is populated through a derivative script, data types are the same both in the rowy table and the firebase collection (number).

**Comment by BeeBombshell** [comment, marked as answer]
Hey @jnnptr, I am assuming there might be some logical error in your Derivative Script. Could you share your Derivative Script here so that we can help you out?

**Comment by jnnptr** [comment]
Hi Bhavya,

It was a difference between the name of the column in firebase and the name
of the key referenced in the column in rowy (a capital letter)

I wanted to delete the entry in GitHub but I couldn't.

Thanks for your support
JP
